diff --git a/node_modules/hexo-asset-file-carp/index.js b/node_modules/hexo-asset-file-carp/index.js
index bc2e636..6be6b29 100644
--- a/node_modules/hexo-asset-file-carp/index.js
+++ b/node_modules/hexo-asset-file-carp/index.js
@@ -133,9 +133,12 @@ hexo.extend.filter.register('after_post_render', function (data) {
            * 不处理网络链接
            * 不处理绝对路径
            */
+          console.info && console.info("update link as:-->", !/http[s]*.*|\/\/.*/.test(src) &&
+            // \s*：匹配零个或多个空白字符。
+            !/^\s*\//.test(src))
           if (!/http[s]*.*|\/\/.*/.test(src) &&
             // \s*：匹配零个或多个空白字符。
-            !/^\s*\//.test(src)) {
+            /^\s*\//.test(src)) {
             // For "about" page, the first part of "src" can't be removed.
             // 关于 "about" 页面，"src" 的第一个部分不能被移除。
             // In addition, to support multi-level local directory.
@@ -157,11 +160,11 @@ hexo.extend.filter.register('after_post_render', function (data) {
             //   srcArray.shift();
             src = srcArray.join('/');
             $(this).attr('src', config.root + link + src);
-            // console.info && // console.info("update link as:-->" + config.root + link + src);
+            console.info && console.info("update link as:-->" + config.root + link + src);
           }
         } else {
-          // console.info && // console.info("no src attr, skipped...");
-          // console.info && // console.info($(this));
+          console.info && console.info("no src attr, skipped...");
+          console.info && console.info($(this));
         }
       });
       data[key] = $.html();
